docs/bokeh/source/docs/user_guide/server/deploy.rst

```diff
@@ -17,6 +17,52 @@ requirements and the expected number of users.
 However, if you have authentication, scaling, or uptime requirements, you'll
 have to consider more sophisticated deployment configurations.
 
+Integrating Bokeh server into other web services
+------------------------------------------------
+
+Bokeh server commonly acts as a solution for constructing plots and
+dashboards and seamlessly embedding them within a larger parent application.
+To accomplish this, the ``bokeh.embed`` module offers the
+:func:`~bokeh.embed.server_document` and :func:`~bokeh.embed.server_session`
+methods. See :ref:`ug_output_embed_apps` for a detailed discussion of their
+usage, with examples.
+
+If the parent service you wish to integrate with is not Python-based, you can
+still integrate with Bokeh through the ``server_document`` / ``server_session``
+methods, but you will need to do so by calling out to a small, long-lived
+Python script that returns the text contents of those methods via any standard
```

Comment:
I don't really understand this scenario fully, would it be possible to give an abbreviated but concrete example? i.e. "parent service" could mean very many things but I am not sure exactly what it refers to here.